src/components/Footer/FooterPrimary/StyledFooterPrimary.js

```diff
@@ -1,18 +1,78 @@
 import styled from 'styled-components'
 import { resolveMedia } from '../../../utils/helpers'
 import { Row, Col } from '../../Grid'
+import { Anchor } from '../../Anchor'
+import { Paragraph } from '../../Paragraph'
 
+// row container
 const FooterPrimaryRow = styled(Row)`
   ${props =>
     !props.background && `background-color: ${props.theme.colors.alternate};`}
-  ${props => !props.padding && `padding: 24px 0 12px;`}
+  ${props => !props.padding && `padding: 24px 15px 12px;`}
 `
 
-const SocialMediaLinks = styled(Col)``
+// social media section
+const SocialContainer = styled(Col)`
+  ${props => !props.padding && `padding: 12px 0;`}
```

Comment:
I think I would make the value of `justify-content` to be `space-around` here to give the social media icons a bit more breathing space.